.npmrc

```diff
@@ -1 +1,2 @@
-save-exact = false
\ No newline at end of file
+save-exact = false
+scripts-prepend-node-path = true
```

Comment:
Fix for:

![image](https://user-images.githubusercontent.com/1477453/79857614-b91ac200-83d6-11ea-9cad-73400d95faf2.png)